src/scrconf/dns_named.scr

```diff
@@ -22,7 +22,7 @@
 	    "comments"	: [ "^[ \t]*#.*$", "^[ \t]*$" ],
 	    "params"    : [
 		$[
-		    "match" : [ "([^ \t]+)[ \t]([^ \t]+(.*[^ \t]+)+)[ \t]*;$", "%s %s;" ],
+		    "match" : [ "([^ \t]+)[ \t]([^ \t]+(.*[^ \t]+)+)[ \t]*;", "%s %s;" ],
```

Comment:
I tried a few scenarios and the solution that even would cope with comments after the semicolon would be:
[ \t];.*$
Both of the above versions fail on that and also the current implementation.
So with this one I would even fix a not-yet-found bug - although the parser needs improvement anyway.